app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  has_many :comments, class_name: 'Comment', foreign_key: 'post_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'post_id'
+
+  belongs_to :user, class_name: 'User', foreign_key: 'users_id'
```

Comment:
Please add association between an `author` and a post so a post belongs to an `author`, not a user.
`belongs_to :author, class_name: 'User'
`